src/Conf.php

```diff
@@ -29,9 +29,9 @@ class Conf
     /**
      * Create new instance of Conf class.
      */
-    public function __construct()
+    public function __construct($filepath)
     {
-        $this->file = storage_path('app/conf.json');
+        $this->file = ($filepath) ? $filepath : storage_path('app/conf.json');
```

Comment:
But as Laravel package, this constructor not supposed to be instantiated manually.
Maybe better to add setter, so you can set the file param. Anyway I think finally it should be set in configuration, where to put the file.
So, give me 10 minutes, I'll make the setter as a temporary solution.